LOGIC_GATES/CONDITIONALS/CONDITIONAL/CONDITIONAL.py

```diff
@@ -29,8 +29,7 @@ def CONDITIONAL(
     # y_of_x = x.y
     # y_of_y = y.y
 
-    bool_ = compare_values(x.c, y.c, operator_type)
-
+    bool_ = compare_values(x.c[0], y.c[0], operator_type)
```

Comment:
Why indexing? If this is needed, something is wrong.. :thinking:

Reply:
Conditional node broke when comparing to Scalar types of different lengths (the usual "the truth value of an array with more than one element is ambiguous" error), despite the fact that the functionality should be defined

Reply:
Scalar shouldn't be an array at all though? It should just be a float

Reply:
To illustrate the exact behaviour, without these changes, the loop below will just restart the loop and not follow either of the bodies connected to the conditional node. Only by the changes above will the conditional node actually redirect the workflow to its children branches.

[Screencast from 08-26-2023 01:50:26 PM.webm](https://github.com/flojoy-ai/nodes/assets/53545754/6af8f997-9be0-48aa-8a7e-db1d2e748dee)

Reply:
Fixed by this PR: https://github.com/flojoy-ai/python/pull/83

Reply:
Done! Will merge the PR now.